plugin/init.lua

```diff
@@ -105,24 +114,30 @@ pub.encryption = {
 	private_key = nil,
 	public_key = nil,
 	encrypt = function(file_path, lines)
+		wezterm.emit("resurrect.encrypt.start", file_path)
 		local cmd = string.format("age -r %s -o %s", pub.encryption.public_key, file_path:gsub(" ", "\\ "))
 		local success, output = execute_cmd_with_stdin(cmd, lines)
 
 		if not success then
+			wezterm.emit("resurrect.error", "Encryption failed: " .. output)
 			wezterm.log_error("Encryption failed: " .. output)
 			return
```

Comment:
Would it be better to error here instead of return and then handle the error in the function which calls this?

Reply:
Moved in latest push